variables/typography.scss

```diff
@@ -24,65 +24,136 @@
   @return map-get($typography-map, size);
 }
 
-$font-family: 'Roboto', Arial ;
- 
-// Klassene som legges på .is- i typography i utilities
+$font-family: 'Roboto', Arial;
+
+:root {
+  --hdd-font-family: #{$font-family};
+}
+
 $typography: (
   h1: (
-    family: $font-family,
-    size: 1.25rem,
-    spacing: .21px,
+    family: var(--hdd-font-family),
+    size: 1.5rem,
     weight: 500,
+    spacing: 0.15px,
     color: var(--hdd-color-font-color),
-    elements: (h1)
+    line-height: 120%,
+    elements: (
+      h1
+    )
   ),
   h2: (
-    family: $font-family,
-    size: 1rem,
-    spacing: .21px,
-    weight: 400,
+    family: var(--hdd-font-family),
+    size: 1.25rem,
+    spacing: 0.15px,
+    weight: normal,
     color: var(--hdd-color-font-color),
-    elements: (h2)
+    line-height: 120%,
+    elements: (
+      h2
+    )
   ),
   h3: (
-    family: $font-family,
-    size: .875rem,
-    spacing: .21px,
+    family: var(--hdd-font-family),
+    size: 1rem,
     weight: 500,
+    spacing: 0.15px,
+    color: var(--hdd-color-font-color),
+    line-height: 120%,
+    elements: (
+      h3
+    )
+  ),
+  h4: (
+    family: var(--hdd-font-family),
+    size: 0.875rem,
+    weight: bold,
+    spacing: 0.5px,
     color: var(--hdd-color-font-color),
-    elements: (h3)
+    line-height: 120%,
+    elements: (
+      h4
+    )
   ),
   p: (
-    family: $font-family,
-    size: .875rem,
-    spacing: .21px,
-    weight: 400,
+    family: var(--hdd-font-family),
+    size: 1rem,
+    weight: normal,
+    spacing: 0.15px,
     color: var(--hdd-color-font-color),
-    elements: (p)
+    line-height: 150%,
```

Comment:
bytt alle til  unitless numbers for line-height values
https://developer.mozilla.org/en-US/docs/Web/CSS/line-height